File: egyeni_sport/model/UserModel.php
<?php

class UserModel{
    private string $felhasznalonev;
    private string $jelszo;
    private string $nev;
    private boolean $admin;

    public function getFelhasznalonev(): string
    {
        return $this->felhasznalonev;
    }

    public function setFelhasznalonev(string $felhasznalonev): void
    {
        $this->felhasznalonev = $felhasznalonev;
    }

    public function getJelszo(): string
    {
        return $this->jelszo;
    }

    public function setJelszo(string $jelszo): void
    {
        $this->jelszo = $jelszo;
    }

    public function getNev(): string
    {
        return $this->nev;
    }

    public function setNev(string $nev): void
    {
        $this->nev = $nev;
    }

    public function isAdmin(): bool
    {
        return $this->admin;
    }

    public function setAdmin(bool $admin): void
    {
        $this->admin = $admin;
    }
}

?>

File: enyeni_sport_spring/src/main/java/hu/egyenisport/enyeni_sport_spring/model/UserModel.java
package hu.egyenisport.enyeni_sport_spring.model;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserModel {
    private String felhasznalonev;
    private String jelszo;
    private String nev;
    private boolean admin;

    public UserModel(String felhasznalonev, String jelszo, String nev, boolean admin) {
        this.felhasznalonev = felhasznalonev;
        this.jelszo = jelszo;
        this.nev = nev;
        this.admin = admin;
    }

    public UserModel(String felhasznalonev) {
        this.felhasznalonev = felhasznalonev;
    }
}
